projects/rero/ng-core/src/lib/record/editor/extensions.ts

```diff
@@ -398,6 +404,22 @@ export class NgCoreFormlyExtension {
       };
     }
   }
+
+  /**
+   * Assigns the default value on the field if it has
+   * a defaultValueExpression definition in the jsonschema.
+   * @param field - FormlyFieldConfig
+   */
+  private _setDefaultValue(field: FormlyFieldConfig): void {
+    if (field.templateOptions?.defaultValueExpression) {
+      const key = String(field.key);
+      if (!(key in field.model && field.model[key] !== null)) {
+        const expression = field.templateOptions.defaultValueExpression;
+        const expressionFn = Function('expression', `return ${expression};`);
+        field.formControl.setValue(expressionFn());
```

Comment:
Maybe it could be 
```javascript
field.formControl.setValue(() => `${exprerssion}`);
```
But maybe not .... I don't know

Reply:
You have to go through `Function` to interpret the text of the expression.